actionview/lib/action_view/railtie.rb

```diff
@@ -38,10 +38,8 @@ class Railtie < Rails::Engine # :nodoc:
     end
 
     initializer "action_view.per_request_digest_cache" do |app|
-      ActiveSupport.on_load(:action_view) do
-        unless ActionView::Resolver.caching?
-          app.executor.to_run ActionView::Digestor::PerExecutionDigestCacheExpiry
-        end
+      unless ActionView::Resolver.caching?
```

Comment:
This will now run before line 35 (above, in the action_view.caching initializer), meaning that ActionView::Resolver does not yet have the value configured by the application.

Reply:
I don't remember if we did fixed this comment already. I don't think so right? The line 71 still needs to run outside of the `on_load` hook. Am I correct?

Reply:
That's correct.  We have not fixed this yet.